chains/degenchain/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+blockExplorers:
+  - apiUrl: https://explorer.degen.tips/api-docs
```

Comment:
is this to the api or docs? noticed this for a few of the chains in here

Reply:
good catch - should be `{baseurl}/api/eth-rpc`